azure-pipelines.yml

```diff
@@ -90,11 +104,11 @@ stages:
 
       - template: .ci/build.yml@components
         parameters:
+          windowsImage: windows-2019
```

Comment:
Shouldn't this be a EO pipeline?

Reply:
@rmarinho - Indeed it should. I attempted to target the EO compliant windows pool and there are build issues that need to be resolved starting with an issue involving the version of MSBuild presumably provided by the EO agent image.  Plus support for targeting the EO pool needs to be added to XamarinComponents. Rather than co-mingle two separate EO efforts into this PR, let's focus on SBOM here.  Then create another PR for the EO pool compliant work, which will depend on a XamarinComponents change: https://github.com/xamarin/XamarinComponents/pull/1331

Reply:
@rmarinho i created a separate PR to address the update to use an execute order (EO) compliant pool.
https://github.com/xamarin/Essentials/pull/1973